bears/java/PMDBear.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class PMDBear(LocalBear, Lint):
+    executable = 'run.sh'
```

Comment:
yeah that's right... just wanted to encourage you to find a way of using a `pmd` binary or so (if existent :P)